Resources/Prototypes/Roles/Jobs/departments.yml

```diff
@@ -89,6 +89,7 @@
   - PrisonGuard ##nyano
   - Prisoner ##nyano
   - Brigmedic
+  - SecurityGuard # Frontier
```

Comment:
Security guard should not show up under NFSD.

Reply:
Pick one:
department-Civilian-description = Perform small helpful tasks to keep the station sane.
department-Command-description = Manage the crew and keep them working efficiently.
department-Engineering-description = Keep the station operational.
department-Medical-description = Keep the crew healthy.
department-Security-description = Keep the peace around the sector.
department-Science-description = Research new technologies and dangerous artifacts.
department-Specific-description = Jobs that not all stations have.

We can also add new one and edit them out, and to be honest, we need to at that point.

Reply:
`department-Security-description` would be most appropriate in this case, as that should show up under "Security" in both role selection and on crew manifests. Might need a different locstring tho, specifically mentioning Frontier Outpost.

Reply:
Out of this PR scope, Let me know if you want to edit this FTL files yourself and make a new PR.